integrationTests/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "commerce-sdk-integration-test",
```

Comment:
Should add internal: false here as this package is never published.

Reply:
@monkey-jeff: internal: false, or internal: true? internal: false would imply it's not internal and should be published.

Reply:
that is correct..wrote this pre-coffee